test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +72,30 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should provide Internet connection for containers using DNS [Feature:Networking-DNS][LinuxOnly]", func() {
```

Comment:
this is an hybrid cluster per the description, I think the LinuxOnly and WindowsOnly tags doesn't apply here